content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -156,6 +156,14 @@ <h3>Load Balancing</h3>
                <p>Next, we’ll do a <code>curl</code> to the exposed IP address and port. Execute the command multiple times:</p>
                <p><code><b>curl http://"$(minikube ip):$NODE_PORT"</b></b></b></code></p>
                <p>We hit a different Pod with every request. This demonstrates that the load-balancing is working.</p>
+               {{< note >}}<p>If you're running minikube with Docker Desktop as the container driver, a minikube tunnel is needed. This is because containers inside Docker Desktop are isolated from your host computer.<br>
+                <p>In a separate terminal window, execute:<br>
+                  <code><b>minikube service kubernetes-bootcamp --url</b></code></p>
+                <p>The output looks like this:
+                  <pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
+                <p>Then use the given URL to access the app:<br>
+                  <code><b>curl 127.0.0.1:51082</b></code></p>
```

Comment:
Is it obvious that this is an example?

Reply:
@sftim In my opinion, the section is worded in a way that makes it clear that this is an example.